sql_scripts/3_cities.sql

```diff
@@ -0,0 +1,567 @@
+CREATE TABLE IF NOT EXISTS cities
+(
+    id SERIAL PRIMARY KEY,
```

Comment:
Consider INTEGER or event deleting this field if it is not used somewhere on purpose. You can make name and state_id as a PRIMARY KEY instead